src/drawbot_skia/path.py

```diff
@@ -51,8 +51,28 @@ def _qCurveToOne(self, pt1, pt2):
     def _closePath(self):
         self.path.close()
 
+    def beginPath(self, identifier=None):
+        self._pointToSegmentPen = PointToSegmentPen(self)
+        self._pointToSegmentPen.beginPath()
+
+    def addPoint(self, point, segmentType=None, smooth=False, name=None, identifier=None, **kwargs):
+        if not hasattr(self, "_pointToSegmentPen"):
+            raise AttributeError("path.beginPath() must be called before the path can be used as a point pen")
+        self._pointToSegmentPen.addPoint(
+            point,
+            segmentType=segmentType,
+            smooth=smooth,
+            name=name,
+            identifier=identifier,
+            **kwargs
+        )
+
     def _endPath(self):
```

Comment:
This needs to become `endPath(self):`
```suggestion
    def endPath(self):
```